Sources/CreateAPI/Generator/Generator+Render.swift

```diff
@@ -57,19 +62,36 @@ extension Generator {
                         contents.append(keys)
                     }
                     if decl.protocols.isDecodable, properties.contains(where: { $0.defaultValue != nil }) {
```

Comment:
This line evaluates far more often now, and I don't think it's using the right heuristic. Would love to get some feedback.